magento/ece-tools
# MAGECLOUD-3486: [Cloud Docker] Add validations to improve error handling

### Description
Removed possibility to provide empty option values like `php bin/ece-tools docker:build --es`
Now services options require some value.
 

### Fixed Issues (if relevant)

https://magento2.atlassian.net/browse/MAGECLOUD-3486

### Manual testing scenarios

1. Run command  `php bin/ece-tools docker:build --es`
2. Command returns an error
```
 The "--es" option requires a value.  
```
3. Repeat with different services.

See [test](https://jira.corp.magento.com/browse/MAGECLOUD-50)

### Contribution checklist
